src/core/text/Text.js

```diff
@@ -175,42 +143,17 @@ Object.defineProperties(Text.prototype, {
         set: function (style)
         {
```

Comment:
The event listener should be cleared each time so that they don't continue to accumulate if the style is set repeatedly: `if (this._style) this._style.removeListener(this._styleListener);`